src/main/java/ui/pages/LoginPage.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package ui.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import utils.PropertiesReader;
+
+/**
+ * Manages Login Page instance.
+ *
+ * @author Alan Escalera.
+ * @version 1.0 17 March 2020.
+ */
+public class LoginPage extends BasePage {
+
+    @FindBy(id = "username")
+    private WebElement userNameField;
+
+    @FindBy(id = "password")
+    private WebElement passwordField;
+
+    @FindBy(id = "Login")
+    private WebElement loginButton;
+
+    /**
+     * Waits until the web element is loaded.
+     */
```

Comment:
This is correct? today the trainer said that the abstract methods in implemented classes are not necessary to document, only in the abstract class